main.go

```diff
@@ -103,6 +104,7 @@ func clear(bin string) {
 	fmt.Println("$ gohr " + bin)
 }
 func init() {
+	testing.Init()
```

Comment:
[FYI]
Avoid the error "flag provided but not defined" when `go test`, which occurs from Go 1.13.